content/docs/apps/experimental/volume-services.md

```diff
@@ -0,0 +1,32 @@
+---
+menu:
+  docs:
+    parent: experimental
+
+title: Volume services
+---
+
+[**This is an experimental feature.**]({{< relref "docs/apps/experimental/experimental.md" >}}) cloud.gov has enabled experimental support for provisioning persistent filesystem volumes for use by customer applications. This experimental service is provided to help test whether cloud.gov is suitable for hosting applications that depend on a durable filesystem to store their state. See below for caveats and notes about production use.
+
+Volumes are brokered and then bound to applications in the same way as our [Redis]({{< relref "docs/services/redis.md">}}), [Elasticsearch]({{< relref "docs/services/elasticsearch24.md">}}), [database]({{< relref "docs/services/relational-database.md">}}), and [blob store]({{< relref "docs/services/s3.md">}}) services. The volume will be available to the application at a path provided in the VCAP_SERVICES environment variable.
+
+## Caveats for this service
+We're making this service available so that customers can explore migrating their filesystem-dependent applications to cloud.gov. It's not intended for production use yet!
+
+* The filesystem service is only available for particular spaces after customers contact cloud.gov support to have it turned on.
+* The filesystem service is *not* highly-available. This means that the brokered volume may be unavailable briefly from time to time as we update cloud.gov.
+* The filesystem service is *not* tuned for high performance. 
+* The filesystem service is backed by NFS, and applications requiring file locking will therefore need to be aware of NFS file locking mechanics. 
+
+All of these points are expected to be addressed before we make the filesystem service available in a fully self-service manner for use by all customers. We encourage use of this service now as a way to determine whether certain applications could run in cloud.gov, but we do not regard is as suitable for supporting a production application. **For this reason, the filesystem service will only be turned on for spaces in sandbox and prototyping oganizations.**
+
+## How to use this service
+
+First [contact support]({{< relref "/docs/help" >}}) to ask that the service be made available. Include
```

Comment:
I like the pattern of embedding an email template in a link to encourage users to ask us well-formed questions, like this: `mailto:cloud-gov-support@gsa.gov?subject=volume service request&body=Org name:\n\nSpace name:\n\nSize of volume:\n`. Just need to url-encode the query parameters for this to work.

Reply:
I don't know any tools for doing that. How do you do it?